Bump FairMQ to v1.4.47

rbx opened the pull request:
Contains @ktf's contribution with the GUI mode for the built-in controller plugin.
Needed for https://github.com/AliceO2Group/AliceO2/pull/8004.

ktf reviewed (approved):
Thanks. Does it change anything else critical besides my "GUI" plugin?

rbx commented:
Nothing critical. Fixed a warning and done some changes to how the test suite is executed (now using mostly ipc instead of tcp). And changed from using `FairMQSomething` to ``fair::mq::Something` in many places, but both are still not available and not (yet) deprecated.
